source/localizable/tutorial/service.md

```diff
@@ -107,7 +107,7 @@ export default Ember.Component.extend({
   didInsertElement() {
     this._super(...arguments);
     let location = this.get('location');
-    let mapElement = this.get('maps').getMapElement(location)
+    let mapElement = this.get('maps').getMapElement(location);
```

Comment:
Thanks!  We actually only need you to make this change (and remove the change from all the other files) as our translations service will handle the updates to the other files.  Would you mind amending your commit?

Thanks!